packages/core/src/components/table/table.css

```diff
@@ -17,8 +17,10 @@
   border-bottom: 1px solid var(--color-black-20);
   white-space: nowrap;
   text-align: left;
-  font-weight: 500;
+  font-weight: 700;
```

Comment:
The font feels almost too bold. In other places the font got thinner but in here it is bolder?

Reply:
Yeah, the S, XS and XXS got bolder (700) in this update: https://helsinkisolutionoffice.atlassian.net/browse/HDS-809. Since the heading in table th was fontsize-heading-xxs, it was set to 700. Same goes for the table caption. I wonder if we should revert these changes?

Reply:
Asked Roni and the table caption and th should not use heading styles, but body styles. So I'll change these back to body styles:

- [ ] Revert table caption and th changes